src/OpenTelemetry.Resources.Azure/OpenTelemetry.Resources.Azure.csproj

```diff
@@ -15,6 +15,8 @@
   <ItemGroup>
     <PackageReference Include="OpenTelemetry" Version="$(OpenTelemetryCoreLatestVersion)" />
     <PackageReference Include="System.Text.Json" Version="4.7.2" />
+    <!-- System.Text.Encodings.Web is indirect reference. It is needed to upgrade it directly to avoid https://github.com/advisories/GHSA-ghhp-997w-qr28 -->
+    <PackageReference Include="System.Text.Encodings.Web" Version="4.7.2" />
```

Comment:
The .NET Standard 2.0 version of `System.Text.Json` has an indirect dependency on `System.Text.Encodings.Web`. It is better to upgrade `System.Text.Json`, as was done in [OpenTelemetry.Exporter.OneCollector](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/4791960d8c49adb9e5b45b8900010e3337c06591/src/OpenTelemetry.Exporter.OneCollector/OpenTelemetry.Exporter.OneCollector.csproj#L27)

Reply:
We may need to re-evaluate whether this is still a challenge with auto-instrumentation - https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1279#issuecomment-1652182987

Reply:
If possible, I would be more conservative with versioning. This solution is applied in Exporter.Console. I do not see any reason we cannot follow this pattern.

Reply:
We will explore it outside of this PR to understand System.Text.Json version could be bumped.